front/app/components/FormBuilder/components/FormBuilderToolbox/ToolboxItem.tsx

```diff
@@ -70,78 +61,78 @@ const ToolboxItem = ({
   }
 
   return (
-    <Tooltip
-      placement={'left-start'}
-      disabled={!disabled || !disabledTooltipMessage}
-      theme={'dark'}
-      content={
-        disabledTooltipMessage && (
-          <Box style={{ cursor: 'default' }}>
-            <Text my="8px" color="white" fontSize="s">
-              {formatMessage(disabledTooltipMessage)}
-            </Text>
-          </Box>
-        )
-      }
+    <Drag
```

Comment:
The toolbox items are wrapped in a drag component to make sure they can work with the drag and drop in the content builder.